core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.html

```diff
@@ -25,8 +25,8 @@
       style="height: 4em;"
       ng-click="$ctrl.editQuestion(question.summary)">
     <td> <[$ctrl.getQuestionIndex($index)]> </td>
-    <td> <[$ctrl.truncatedQuestionSummaries[$index]]> </td>
-    <td ng-if="!$ctrl.getSkill()">{{question.skill_description}}</td>
+    <td> {{question.summary.question_content.slice(3, question.summary.question_content.length - 4)}} </td>
```

Comment:
Dont use` {{param}}` replace it with `<[param]>`
Also this section is a bit confusing, what you did previously (i.e. calling a function that returns this value) is a better idea

Reply:
Done.